components/brave_wallet/browser/swap_controller.cc

```diff
@@ -117,6 +117,24 @@ GURL SwapController::GetTransactionPayloadURL(
   return url;
 }
 
+void SwapController::GetSwapConfiguration(
+    const std::string& chain_id,
+    GetSwapConfigurationCallback callback) {
+  auto swap_config = mojom::SwapConfig::New();
+
+  if (chain_id == mojom::kRopstenChainId) {
+    swap_config->swap_api_url = kRopstenSwapAPIURL;
+    swap_config->buy_token_percantage_fee = kRopstenBuyTokenPercentageFee;
+    swap_config->fee_recipient = kRopstenFeeRecipient;
+  } else {
+    swap_config->swap_api_url = kSwapAPIURL;
+    swap_config->buy_token_percantage_fee = kBuyTokenPercentageFee;
+    swap_config->fee_recipient = kFeeRecipient;
+  }
+
+  std::move(callback).Run(std::move(swap_config));
```

Comment:
It would be easy for here to add a unittest to test both of these cases with expected config here `components/brave_wallet/browser/swap_controller_unittest.cc`.